### ISSUE TITLE
is_prime(::IdealSheaf) buggy

### ISSUE THREAD

**Issue by simonbrandhorst** [oscar-system/Oscar.jl#2479]
In the following I construct the ideal sheaf on $P^1$ consisting of the points `[0:1]` and `[1:0]` 
Oscar claims that this sheaf of ideals is prime.  However it is not. (It is only locally prime). 
Indeed, if we take an affine chart, e.g. $x_0+x_1 \neq 0$,  containing both points the corresponding ideal is not prime. 

```
julia> p1 = projective_space(QQ,1);

julia> U = C[1][1];

julia> V = C[1][2];

julia> I1 = IdealSheaf(C, U, coordinates(U))
Ideal Sheaf on Covered Scheme with 2 Charts

julia> I2 = IdealSheaf(C, V, coordinates(V))
Ideal Sheaf on Covered Scheme with 2 Charts

julia> I1*I2
Ideal Sheaf on Covered Scheme with 2 Charts

julia> is_prime(I1*I2)
true
```

**Comment by simonbrandhorst** [comment]
One solution is to rename the function to `is_lsocally_prime` and to document is accordingly. 
@afkafkafk13  @HechtiDerLachs

**Comment by fingolfin** [comment]
According to @afkafkafk13 there is a PR by @simonbrandhorst (?) where this is being renamed (if someone knows more, please provide the details in a comment). Then future work can add a new correct `is_prime`.

**Comment by simonbrandhorst** [comment]
Fixed in #2508
